src/utils.jl

```diff
@@ -104,22 +104,21 @@ end
     
 Build a vector of `AbstractArray` from `new_w` similar to `Knet.params(model.chain)` or `nested_array`.
 Call iteratively `build_layer_from_vec!` to build each intermediate `AbstractArray`.
-This method is not optimized; it allocates memory.
```

Comment:
leave it if still true

Reply:
It should be true if `copyto!()` doesn't allocate memory.
I keep
```julia
"""
This method is not optimized; it allocates memory.
"""
```
I open an issue on this topic.